instructor/ui/instructor.js

```diff
@@ -180,6 +180,15 @@ $(function(){
         }
     });
     /**
+     * Western Blotting: Antibodies
+     */
+    if($("#id_form-0-primary").length){
```

Comment:
Are these CSS classes specific to the western blotting antibodies page? If not can you add one to prevent this code from executing elsewhere?